exams/pearson/writers.py

```diff
@@ -160,21 +161,60 @@ def __init__(self):
     @classmethod
     def first_name(cls, profile):
         """
-        romanized_first_name if we have it, first_name otherwise
+        Determines which first_name to use
+
+        Args:
+            profile (profiles.models.Profile): the profile to get state data from
+
+        Returns:
+            str: romanized_first_name if we have it, first_name otherwise
         """
         return profile.romanized_first_name or profile.first_name
 
     @classmethod
     def last_name(cls, profile):
         """
-        romanized_last_name if we have it, last_name otherwise
+        Determines which last_name to use
+
+        Args:
+            profile (profiles.models.Profile): the profile to get state data from
+
+        Returns:
+            str: romanized_last_name if we have it, last_name otherwise
         """
         return profile.romanized_last_name or profile.last_name
 
     @classmethod
+    def profile_state(cls, profile):
+        """
+        Transforms the state into the format accepted by PEARSON
+
+        Args:
+            profile (profiles.models.Profile): the profile to get state data from
+
+        Returns:
+            str: 2-character state representation if country is US/CA otherwise None
+        """
+        if profile.country not in PEARSON_STATE_SUPPORTED_COUNTRIES:
+            return ''
+
+        state = profile.state_or_territory
+
+        if '-' in state:
+            _, state = state.split('-')
```

Comment:
i think it makes sense to implement this split as part of `Profile` class. `Profile.territory_state_tuple` or something like that

this could also make sense for `split_phone_number`, but that could be considered for a different PR if you prefer